src/instructions.ts

```diff
@@ -6,15 +6,17 @@ export type Command = {
   args: any[];
 };
 
-export type TypeText = { type: 'typeText'; text: string[] };
+export type Wait = { type: 'wait', length: number | 'manual' };
```

Comment:
Can you change `length` to `delay` here so it is the same as in the `TypeText` command please?